projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -13,6 +13,7 @@
     "cy:run:mobile": "cypress run --spec \"cypress/integration/mobile/**/*\"",
     "cy:run:mobile:ci": "cypress run --config-file cypress.ci.1905.json --spec \"cypress/integration/mobile/**/*\"",
     "cy:run:regression": "cypress run --spec \"cypress/integration/regression/**/*\"",
+    "cy:cds:run:vendor": "cypress run --spec \"cypress/integration/vendor/cds/**/*\"",
```

Comment:
@bgambocjaviniar what do you think, do we need this line? Not sure where it's used, most likely in a script?

Reply:
yup it's being used in the script, and it's best to keep it here to make it easier for them to run it in headless through a single command. I believe this was removed accidentally by someone before, therefore it was brought back here